index.bs

```diff
@@ -1406,7 +1406,7 @@ Instances of <code>WritableStream</code> are created with the internal slots des
 <ol>
   <li> Assert: <var>stream</var>@\[[state]] is <code>"closing"</code>.
   <li> Let <var>sinkClosePromise</var> be PromiseInvokeOrNoop(<var>stream</var>@\[[underlyingSink]],
-    <code>"close"</code>).
+    <code>"close"</code>, «‍»).
```

Comment:
Hmm it seems ES conventions don't include passing through missing parameters, e.g. [Invoke](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-invoke) explicitly sets the arguments list to empty even though Call already does. I'll add those steps I guess.